devel/creduce/Portfile

```diff
@@ -60,7 +62,6 @@ if {${os.platform} eq "darwin" && ${os.major} <= 12} {
     depends_lib-append      port:libcxx
     configure.cxx_stdlib    libc++
     # clang < macOS 10.9 don't work. clang-5.0 tested and works.
-    # cannot use cxx11 1.1 PG due to other issues
```

Comment:
The cxx11 1.1 portgroup had been added anyway sometime after this comment was left. I am removing it since it's irrelevant now; any other work/cleanup suggested?